### ISSUE TITLE
Add Start Command

### ISSUE THREAD

**Issue by mosaid88** [arachnetech/homebridge-mqttthing#31]
StartCmd will allow to get device status on start. This should be triggered at homebridge start

```
"startCmd": "cmnd/lights/POWER1",
"startParameter": ""
```

startParameter is also really handy as it can be used in 2 ways, Empty: will return current switch state, or if value provided (ON or OFF) then it can be used to set a default value on start of homebridge.

**Comment by arachnetech** [comment]
Just confirming functionality: `startCmd` is the MQTT topic and `startParameter` is the message to be published to that topic on startup?

Presumably used as 'empty' it simply triggers the accessory to publish current state to appropriate getXXX topics where homebridge-mqttthing would pick it up as usual?

If used to set a default value on start of homebridge, what about when the accessory uses multiple topics (e.g. one for a light on/off, another for colour)?

**Comment by matej** [comment]
> If used to set a default value on start of homebridge, what about when the accessory uses multiple topics (e.g. one for a light on/off, another for colour)?

Right. That would be a limitation here. Perhaps taking an array of those would be more flexible?

**Comment by NorthernMan54** [comment]
@mientki Thanks for pointing out this feature, I went and played with my Wifi dimmer's and figured out how to make it work.  I have updated my settings on the wiki that you can use as an example.

https://github.com/arachnetech/homebridge-mqttthing/wiki/Tested-Configurations#ce-smart-wifi-smart-dimmer

**Comment by NorthernMan54** [comment]
@mientki I had the same problem you mentioned, and fixed it with the config on the wiki.

**Comment by mientki** [comment]
can you tell me more? which parameter is responsible for it? can you buy show your config one accessory?

**Comment by arachnetech** [comment]
It's at https://github.com/arachnetech/homebridge-mqttthing/wiki/Tested-Configurations#ce-smart-wifi-smart-dimmer

MQTT messages may be published on start-up, e.g. to reset accessories to a known initial state, with `startPub`. This is an object containing MQTT topics as keys, and values to be published as values.

**Comment by alesoblak** [comment]
> I can not publish the ON or OFF topic after the HomeBridge restart. The subject must be subscribe from the broker or device. The way it is. No one else has such a problem after HomeBridge restart? Do you have the right device states after the restart? Please, show me your usual light bulb configurations.

Did you managed to solve this? I have the same problem. After homebridge restarts all mqtt lights and switches are shown as OFF. I cant figure it out.

**Comment by damianmroz** [comment]
I heve the same issue. Configured everything as in Tested configuration: https://github.com/arachnetech/homebridge-mqttthing/wiki/Tested-Configurations#ce-smart-wifi-smart-dimmer
Still accessories after restarting homebridge are all shown as OFF.

**Comment by NorthernMan54** [comment]
Can your share your config.json fragment.

**Comment by damianmroz** [comment]
For now my config looks like this.
I moved to hoobs in the meantime and this is only way i can add this accessory.
Still not working ;/
```
{
    "name": "Living Room",
    "onValue": "ON",
    "onlineValue": "Online",
    "username": "xxx",
    "url": "mqtt://192.168.1.30",
    "type": "lightbulb",
    "topics": {
        "getOn": "stat/livingroom/POWER",
        "setOn": "cmnd/livingroom/power",
        "startPub": "cmnd/livingroom/power"
    },
    "accessory": "mqttthing",
    "password": "xxx",
    "offValue": "OFF"
}
```

**Comment by NorthernMan54** [comment]
Look at the sample carefully and your config, they are not the same
